docs/source/make_uploader_docs.py

```diff
@@ -0,0 +1,40 @@
+from argparse import ArgumentParser
+import imp
+import os
+
+
+template = """
+{name}
```

Comment:
Maybe add an rst-formatted comment here saying "This part of the document (down to the end of the 'Usage' section) is auto-generated by a script"